### PR TITLE
Feat/dns order

### PR DESCRIPTION
**Pull request by marie-j**
### Description of the Change

Replace legacy call with Agora order 
MBP-251

Will rebase once https://github.com/ovh-ux/ovh-manager-web/pull/1201 is merged

### CODE REVIEW

**Comment by antleblanc** [comment]
@marie-j #1201 has been merged.

Feel free to rebase on top of `develop` branch if needed.

Thanks!

**Comment by marie-j** [comment]
> @marie-j #1201 has been merged.
> 
> Feel free to rebase on top of `develop` branch if needed.
> 
> Thanks!

Rebase on `develop` has been made

File: client/app/domain/zone/domain-zone-dns.controller.js [hunk 1]
```diff
@@ -1,242 +1,241 @@
...
-      this.User = User;
-    }
-
-    $onInit() {
-      this.domain = this.$scope.ctrlDomain.domain;
-
-      this.allowModification = false;
-      this.atLeastOneSelected = false;
+export default class DomainTabZoneDnsCtrl {
+  constructor($scope, $q, $translate, activateZone, Alerter, Domain, User) {
```
Comment (antleblanc) [inline client/app/domain/zone/domain-zone-dns.controller.js]:
```suggestion
  /* @ngInject */
  constructor($scope, $q, $translate, activateZone, Alerter, Domain, User) {
```

**Comment by jleveugle** [comment]
Hello @marie-j,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Pull request merged**